src/dispatch/nlp.py

```diff
@@ -0,0 +1,52 @@
+import logging
```

Comment:
Should we create a directory for nlp and rename this module to service?

Reply:
Not yet, I don't think. A dir with one file seems like overkill, i.e I don't see nlp ever needing view or models.